README.md

```diff
@@ -60,60 +60,12 @@ Configuration example in `mysite/_config/spamprotection.yml`
 
 Then once you have setup this config you will need to include the spam protector field as per the instructions on the [spamprotection](https://github.com/silverstripe/silverstripe-spamprotection) page.
 
-
-## Custom Theming
```

Comment:
Why has this been removed? It's still a feature of the API: https://developers.google.com/recaptcha/old/docs/customization#custom-theming